### PR TITLE
Rs/create custom audioplayer

### PR DESCRIPTION
**Pull request by rccsousa**
Why:
- podcast episode pages had the default browser audioplayer, which was not intended;
How:
- added a custom audioplayer for both desktop and mobile versions of the podcast episode page;

imagem
imagem

### CODE REVIEW

**Comment by rccsousa** [comment]
@davelange i'm experimenting a somewhat different approach in terms of sub-components on this one. since the `AudioPlayer` is exclusive to podcasts it felt more logical to encapsulate it within the EpisodeHead. i'd like to hear your thoughts on this, if it makes sense i'll add this structure to the planned refactor down the road. thanks in advance!

File: src/app/_blocks/EpisodeHead/AudioPlayer/ButtonIcons/index.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
+export function AudioPlayButton({ width = '80', height = '54' }) {
```
Comment (davelange) [inline src/app/_blocks/EpisodeHead/AudioPlayer/ButtonIcons/index.tsx:1]: Can we rename this to AudioPlayIcon? Just to distinguish it from actual buttons

**Review by davelange — APPROVED** [review]
This is very nice 🚀

File: src/app/_blocks/EpisodeHead/AudioPlayer/ButtonIcons/index.tsx (hunk 1 above)
Comment (rccsousa) [reply src/app/_blocks/EpisodeHead/AudioPlayer/ButtonIcons/index.tsx:1]: Off course, makes total sense. Will update it before merging.

**Pull request merged**